### ISSUE TITLE
ext/XS-APItest/t/boolean*.t: new test failures with PERLIO=stdio

### ISSUE THREAD

**Issue by jkeenan** [Perl/perl5#19780]
I am observing new test failures in `ext/XS-APItest/t/boolean-thr.t` and `ext/XS-APItest/t/boolean.t` when those program are run through the harness with `PERLIO=stdio`.

As seen on FreeBSD-13, threaded build, at v5.37.0-49-ge1980ad278:
```
$ cd t;TEST_JOBS=1 PERLIO=stdio ./perl harness -v ../ext/XS-APItest/t/boolean*.t;
cd -
../ext/XS-APItest/t/boolean-thr.t .. 
ok 1 - value in to thread is bool
ok 2 - value out of thread is bool
ok 3 - variable captured by thread is bool
ok 4 - :shared variable is bool outside
ok 5 - :shared variable is bool inside thread
# ok: SvIOK(sviv) && !SvIandPOK(sviv)
# ok: SvPOK(svpv) && !SvIandPOK(svpv)
# ok: SvIOK(sviv) && !SvBoolFlagsOK(sviv)
# ok: SvPOK(svpv) && !SvBoolFlagsOK(svpv)
# Testing 'newSVsv(&PL_sv_yes) and newSVsv(&PL_sv_no)'
# ok: SvCUR(true_sv) == 1
# ok: SvCUR(false_sv) == 0
# ok: SvLEN(true_sv) == 0
# ok: SvLEN(false_sv) == 0
# ok: SvIV(true_sv) == 1
# ok: SvIV(false_sv) == 0
# ok: SvIsCOW(true_sv)
# ok: SvIsCOW(false_sv)
# ok: strEQ(SvPV_nolen(true_sv),"1")
# ok: strEQ(SvPV_nolen(false_sv),"")
# ok: SvIOK(true_sv)
# ok: SvIOK(false_sv)
# ok: SvPOK(true_sv)
# ok: SvPOK(false_sv)
# ok: SvBoolFlagsOK(true_sv)
# ok: SvBoolFlagsOK(false_sv)
# ok: SvTYPE(true_sv) >= SVt_PVNV
# ok: SvTYPE(false_sv) >= SVt_PVNV
# ok: SvBoolFlagsOK(true_sv) && BOOL_INTERNALS_sv_isbool(true_sv)
# ok: SvBoolFlagsOK(false_sv) && BOOL_INTERNALS_sv_isbool(false_sv)
# ok: SvBoolFlagsOK(true_sv) && BOOL_INTERNALS_sv_isbool_true(true_sv)
# ok: SvBoolFlagsOK(false_sv) && BOOL_INTERNALS_sv_isbool_false(false_sv)
# ok: SvBoolFlagsOK(true_sv) && !BOOL_INTERNALS_sv_isbool_false(true_sv)
# ok: SvBoolFlagsOK(false_sv) && !BOOL_INTERNALS_sv_isbool_true(false_sv)
# ok: SvTRUE(true_sv)
# ok: !SvTRUE(false_sv)
# Testing 'newSV_true() and newSV_false()'
# ok: SvCUR(true_sv) == 1
# ok: SvCUR(false_sv) == 0
# ok: SvLEN(true_sv) == 0
# ok: SvLEN(false_sv) == 0
# ok: SvIV(true_sv) == 1
# ok: SvIV(false_sv) == 0
# ok: SvIsCOW(true_sv)
# ok: SvIsCOW(false_sv)
# ok: strEQ(SvPV_nolen(true_sv),"1")
# ok: strEQ(SvPV_nolen(false_sv),"")
# ok: SvIOK(true_sv)
# ok: SvIOK(false_sv)
# ok: SvPOK(true_sv)
# ok: SvPOK(false_sv)
# ok: SvBoolFlagsOK(true_sv)
# ok: SvBoolFlagsOK(false_sv)
# ok: SvTYPE(true_sv) >= SVt_PVNV
# ok: SvTYPE(false_sv) >= SVt_PVNV
# ok: SvBoolFlagsOK(true_sv) && BOOL_INTERNALS_sv_isbool(true_sv)
# ok: SvBoolFlagsOK(false_sv) && BOOL_INTERNALS_sv_isbool(false_sv)
# ok: SvBoolFlagsOK(true_sv) && BOOL_INTERNALS_sv_isbool_true(true_sv)
# ok: SvBoolFlagsOK(false_sv) && BOOL_INTERNALS_sv_isbool_false(false_sv)
# ok: SvBoolFlagsOK(true_sv) && !BOOL_INTERNALS_sv_isbool_false(true_sv)
# ok: SvBoolFlagsOK(false_sv) && !BOOL_INTERNALS_sv_isbool_true(false_sv)
# ok: SvTRUE(true_sv)
# ok: !SvTRUE(false_sv)
# Testing 'newSVbool(1) and newSVbool(0)'
# ok: SvCUR(true_sv) == 1
# ok: SvCUR(false_sv) == 0
# ok: SvLEN(true_sv) == 0
# ok: SvLEN(false_sv) == 0
# ok: SvIV(true_sv) == 1
# ok: SvIV(false_sv) == 0
# ok: SvIsCOW(true_sv)
# ok: SvIsCOW(false_sv)
# ok: strEQ(SvPV_nolen(true_sv),"1")
# ok: strEQ(SvPV_nolen(false_sv),"")
# ok: SvIOK(true_sv)
# ok: SvIOK(false_sv)
# ok: SvPOK(true_sv)
# ok: SvPOK(false_sv)
# ok: SvBoolFlagsOK(true_sv)
# ok: SvBoolFlagsOK(false_sv)
# ok: SvTYPE(true_sv) >= SVt_PVNV
# ok: SvTYPE(false_sv) >= SVt_PVNV
# ok: SvBoolFlagsOK(true_sv) && BOOL_INTERNALS_sv_isbool(true_sv)
# ok: SvBoolFlagsOK(false_sv) && BOOL_INTERNALS_sv_isbool(false_sv)
# ok: SvBoolFlagsOK(true_sv) && BOOL_INTERNALS_sv_isbool_true(true_sv)
# ok: SvBoolFlagsOK(false_sv) && BOOL_INTERNALS_sv_isbool_false(false_sv)
# ok: SvBoolFlagsOK(true_sv) && !BOOL_INTERNALS_sv_isbool_false(true_sv)
# ok: SvBoolFlagsOK(false_sv) && !BOOL_INTERNALS_sv_isbool_true(false_sv)
# ok: SvTRUE(true_sv)
# ok: !SvTRUE(false_sv)
# Testing 'true_sv_setsv and false_sv_setsv'
# ok: SvCUR(true_sv) == 1
# ok: SvCUR(false_sv) == 0
# ok: SvLEN(true_sv) == 0
# ok: SvLEN(false_sv) == 0
# ok: SvIV(true_sv) == 1
# ok: SvIV(false_sv) == 0
# ok: SvIsCOW(true_sv)
# ok: SvIsCOW(false_sv)
# ok: strEQ(SvPV_nolen(true_sv),"1")
# ok: strEQ(SvPV_nolen(false_sv),"")
# ok: SvIOK(true_sv)
# ok: SvIOK(false_sv)
# ok: SvPOK(true_sv)
# ok: SvPOK(false_sv)
# ok: SvBoolFlagsOK(true_sv)
# ok: SvBoolFlagsOK(false_sv)
# ok: SvTYPE(true_sv) >= SVt_PVNV
# ok: SvTYPE(false_sv) >= SVt_PVNV
# ok: SvBoolFlagsOK(true_sv) && BOOL_INTERNALS_sv_isbool(true_sv)
# ok: SvBoolFlagsOK(false_sv) && BOOL_INTERNALS_sv_isbool(false_sv)
# ok: SvBoolFlagsOK(true_sv) && BOOL_INTERNALS_sv_isbool_true(true_sv)
# ok: SvBoolFlagsOK(false_sv) && BOOL_INTERNALS_sv_isbool_false(false_sv)
# ok: SvBoolFlagsOK(true_sv) && !BOOL_INTERNALS_sv_isbool_false(true_sv)
# ok: SvBoolFlaok 6 - Bulk test internal bool related APIs
1..6
gsOK(false_sv) && !BOOL_INTERNALS_sv_isbool_true(false_sv)
# ok: SvTRUE(true_sv)
# ok: !SvTRUE(false_sv)
# Testing 'true_sv_set_true and false_sv_set_false'
# ok: SvCUR(true_sv) == 1
# ok: SvCUR(false_sv) == 0
# ok: SvLEN(true_sv) == 0
# ok: SvLEN(false_sv) == 0
# ok: SvIV(true_sv) == 1
# ok: SvIV(false_sv) == 0
# ok: SvIsCOW(true_sv)
# ok: SvIsCOW(false_sv)
# ok: strEQ(SvPV_nolen(true_sv),"1")
# ok: strEQ(SvPV_nolen(false_sv),"")
# ok: SvIOK(true_sv)
# ok: SvIOK(false_sv)
# ok: SvPOK(true_sv)
# ok: SvPOK(false_sv)
# ok: SvBoolFlagsOK(true_sv)
# ok: SvBoolFlagsOK(false_sv)
# ok: SvTYPE(true_sv) >= SVt_PVNV
# ok: SvTYPE(false_sv) >= SVt_PVNV
# ok: SvBoolFlagsOK(true_sv) && BOOL_INTERNALS_sv_isbool(true_sv)
# ok: SvBoolFlagsOK(false_sv) && BOOL_INTERNALS_sv_isbool(false_sv)
# ok: SvBoolFlagsOK(true_sv) && BOOL_INTERNALS_sv_isbool_true(true_sv)
# ok: SvBoolFlagsOK(false_sv) && BOOL_INTERNALS_sv_isbool_false(false_sv)
# ok: SvBoolFlagsOK(true_sv) && !BOOL_INTERNALS_sv_isbool_false(true_sv)
# ok: SvBoolFlagsOK(false_sv) && !BOOL_INTERNALS_sv_isbool_true(false_sv)
# ok: SvTRUE(true_sv)
# ok: !SvTRUE(false_sv)
# Testing 'true_sv_set_bool and false_sv_set_bool'
# ok: SvCUR(true_sv) == 1
# ok: SvCUR(false_sv) == 0
# ok: SvLEN(true_sv) == 0
# ok: SvLEN(false_sv) == 0
# ok: SvIV(true_sv) == 1
# ok: SvIV(false_sv) == 0
# ok: SvIsCOW(true_sv)
# ok: SvIsCOW(false_sv)
# ok: strEQ(SvPV_nolen(true_sv),"1")
# ok: strEQ(SvPV_nolen(false_sv),"")
# ok: SvIOK(true_sv)
# ok: SvIOK(false_sv)
# ok: SvPOK(true_sv)
# ok: SvPOK(false_sv)
# ok: SvBoolFlagsOK(true_sv)
# ok: SvBoolFlagsOK(false_sv)
# ok: SvTYPE(true_sv) >= SVt_PVNV
# ok: SvTYPE(false_sv) >= SVt_PVNV
# ok: SvBoolFlagsOK(true_sv) && BOOL_INTERNALS_sv_isbool(true_sv)
# ok: SvBoolFlagsOK(false_sv) && BOOL_INTERNALS_sv_isbool(false_sv)
# ok: SvBoolFlagsOK(true_sv) && BOOL_INTERNALS_sv_isbool_true(true_sv)
# ok: SvBoolFlagsOK(false_sv) && BOOL_INTERNALS_sv_isbool_false(false_sv)
# ok: SvBoolFlagsOK(true_sv) && !BOOL_INTERNALS_sv_isbool_false(true_sv)
# ok: SvBoolFlagsOK(false_sv) && !BOOL_INTERNALS_sv_isbool_true(false_sv)
# ok: SvTRUE(true_sv)
# ok: !SvTRUE(false_sv)
Failed 1/6 subtests 
../ext/XS-APItest/t/boolean.t ...... 
ok 1 - false is boolean
ok 2 - true is boolean
ok 3 - 0 is not boolean
ok 4 - 1 is not boolean
ok 5 - "" is not boolean
ok 6 - false var is boolean
ok 7 - true var is boolean
ok 8 - stringified true is not boolean
ok 9 - numified true is not boolean
ok 10 - true var remains boolean after stringification and numification
ok 11 - false HELEM is boolean
ok 12 - true HELEM is boolean
# ok: SvIOK(sviv) && !SvIandPOK(sviv)
# ok: SvPOK(svpv) && !SvIandPOK(svpv)
# ok: SvIOK(sviv) && !SvBoolFlagsOK(sviv)
# ok: SvPOK(svpv) && !SvBoolFlagsOK(svpv)
# Testing 'newSVsv(&PL_sv_yes) and newSVsv(&PL_sv_no)'
# ok: SvCUR(true_sv) == 1
# ok: SvCUR(false_sv) == 0
# ok: SvLEN(true_sv) == 0
# ok: SvLEN(false_sv) == 0
# ok: SvIV(true_sv) == 1
# ok: SvIV(false_sv) == 0
# ok: SvIsCOW(true_sv)
# ok: SvIsCOW(false_sv)
# ok: strEQ(SvPV_nolen(true_sv),"1")
# ok: strEQ(SvPV_nolen(false_sv),"")
# ok: SvIOK(true_sv)
# ok: SvIOK(false_sv)
# ok: SvPOK(true_sv)
# ok: SvPOK(false_sv)
# ok: SvBoolFlagsOK(true_sv)
# ok: SvBoolFlagsOK(false_sv)
# ok: SvTYPE(true_sv) >= SVt_PVNV
# ok: SvTYPE(false_sv) >= SVt_PVNV
# ok: SvBoolFlagsOK(true_sv) && BOOL_INTERNALS_sv_isbool(true_sv)
# ok: SvBoolFlagsOK(false_sv) && BOOL_INTERNALS_sv_isbool(false_sv)
# ok: SvBoolFlagsOK(true_sv) && BOOL_INTERNALS_sv_isbool_true(true_sv)
# ok: SvBoolFlagsOK(false_sv) && BOOL_INTERNALS_sv_isbool_false(false_sv)
# ok: SvBoolFlagsOK(true_sv) && !BOOL_INTERNALS_sv_isbool_false(true_sv)
# ok: SvBoolFlagsOK(false_sv) && !BOOL_INTERNALS_sv_isbool_true(false_sv)
# ok: SvTRUE(true_sv)
# ok: !SvTRUE(false_sv)
# Testing 'newSV_true() and newSV_false()'
# ok: SvCUR(true_sv) == 1
# ok: SvCUR(false_sv) == 0
# ok: SvLEN(true_sv) == 0
# ok: SvLEN(false_sv) == 0
# ok: SvIV(true_sv) == 1
# ok: SvIV(false_sv) == 0
# ok: SvIsCOW(true_sv)
# ok: SvIsCOW(false_sv)
# ok: strEQ(SvPV_nolen(true_sv),"1")
# ok: strEQ(SvPV_nolen(false_sv),"")
# ok: SvIOK(true_sv)
# ok: SvIOK(false_sv)
# ok: SvPOK(true_sv)
# ok: SvPOK(false_sv)
# ok: SvBoolFlagsOK(true_sv)
# ok: SvBoolFlagsOK(false_sv)
# ok: SvTYPE(true_sv) >= SVt_PVNV
# ok: SvTYPE(false_sv) >= SVt_PVNV
# ok: SvBoolFlagsOK(true_sv) && BOOL_INTERNALS_sv_isbool(true_sv)
# ok: SvBoolFlagsOK(false_sv) && BOOL_INTERNALS_sv_isbool(false_sv)
# ok: SvBoolFlagsOK(true_sv) && BOOL_INTERNALS_sv_isbool_true(true_sv)
# ok: SvBoolFlagsOK(false_sv) && BOOL_INTERNALS_sv_isbool_false(false_sv)
# ok: SvBoolFlagsOK(true_sv) && !BOOL_INTERNALS_sv_isbool_false(true_sv)
# ok: SvBoolFlagsOK(false_sv) && !BOOL_INTERNALS_sv_isbool_true(false_sv)
# ok: SvTRUE(true_sv)
# ok: !SvTRUE(false_sv)
# Testing 'newSVbool(1) and newSVbool(0)'
# ok: SvCUR(true_sv) == 1
# ok: SvCUR(false_sv) == 0
# ok: SvLEN(true_sv) == 0
# ok: SvLEN(false_sv) == 0
# ok: SvIV(true_sv) == 1
# ok: SvIV(false_sv) == 0
# ok: SvIsCOW(true_sv)
# ok: SvIsCOW(false_sv)
# ok: strEQ(SvPV_nolen(true_sv),"1")
# ok: strEQ(SvPV_nolen(false_sv),"")
# ok: SvIOK(true_sv)
# ok: SvIOK(false_sv)
# ok: SvPOK(true_sv)
# ok: SvPOK(false_sv)
# ok: SvBoolFlagsOK(true_sv)
# ok: SvBoolFlagsOK(false_sv)
# ok: SvTYPE(true_sv) >= SVt_PVNV
# ok: SvTYPE(false_sv) >= SVt_PVNV
# ok: SvBoolFlagsOK(true_sv) && BOOL_INTERNALS_sv_isbool(true_sv)
# ok: SvBoolFlagsOK(false_sv) && BOOL_INTERNALS_sv_isbool(false_sv)
# ok: SvBoolFlagsOK(true_sv) && BOOL_INTERNALS_sv_isbool_true(true_sv)
# ok: SvBoolFlagsOK(false_sv) && BOOL_INTERNALS_sv_isbool_false(false_sv)
# ok: SvBoolFlagsOK(true_sv) && !BOOL_INTERNALS_sv_isbool_false(true_sv)
# ok: SvBoolFlagsOK(false_sv) && !BOOL_INTERNALS_sv_isbool_true(false_sv)
# ok: SvTRUE(true_sv)
# ok: !SvTRUE(false_sv)
# Testing 'true_sv_setsv and false_sv_setsv'
# ok: SvCUR(true_sv) == 1
# ok: SvCUR(false_sv) == 0
# ok: SvLEN(true_sv) == 0
# ok: SvLEN(false_sv) == 0
# ok: SvIV(true_sv) == 1
# ok: SvIV(false_sv) == 0
# ok: SvIsCOW(true_sv)
# ok: SvIsCOW(false_sv)
# ok: strEQ(SvPV_nolen(true_sv),"1")
# ok: strEQ(SvPV_nolen(false_sv),"")
# ok: SvIOK(true_sv)
# ok: SvIOK(false_sv)
# ok: SvPOK(true_sv)
# ok: SvPOK(false_sv)
# ok: SvBoolFlagsOK(true_sv)
# ok: SvBoolFlagsOK(false_sv)
# ok: SvTYPE(true_sv) >= SVt_PVNV
# ok: SvTYPE(false_sv) >= SVt_PVNV
# ok: SvBoolFlagsOK(true_sv) && BOOL_INTERNALS_sv_isbool(true_sv)
# ok: SvBoolFlagsOK(false_sv) && BOOL_INTERNALS_sv_isbool(false_sv)
# ok: SvBoolFlagsOK(true_sv) && BOOL_INTERNALS_sv_isbool_true(true_sv)
# ok: SvBoolFlagsOK(false_sv) && BOOL_INTERNALS_sv_isbool_false(false_sv)
# ok: SvBoolFlagsOK(true_sv) && !BOOL_INTERNALS_sv_isbool_false(true_sv)
# ok: SvBoolFlaok 13 - Bulk test internal bool related APIs
1..13
gsOK(false_sv) && !BOOL_INTERNALS_sv_isbool_true(false_sv)
# ok: SvTRUE(true_sv)
# ok: !SvTRUE(false_sv)
# Testing 'true_sv_set_true and false_sv_set_false'
# ok: SvCUR(true_sv) == 1
# ok: SvCUR(false_sv) == 0
# ok: SvLEN(true_sv) == 0
# ok: SvLEN(false_sv) == 0
# ok: SvIV(true_sv) == 1
# ok: SvIV(false_sv) == 0
# ok: SvIsCOW(true_sv)
# ok: SvIsCOW(false_sv)
# ok: strEQ(SvPV_nolen(true_sv),"1")
# ok: strEQ(SvPV_nolen(false_sv),"")
# ok: SvIOK(true_sv)
# ok: SvIOK(false_sv)
# ok: SvPOK(true_sv)
# ok: SvPOK(false_sv)
# ok: SvBoolFlagsOK(true_sv)
# ok: SvBoolFlagsOK(false_sv)
# ok: SvTYPE(true_sv) >= SVt_PVNV
# ok: SvTYPE(false_sv) >= SVt_PVNV
# ok: SvBoolFlagsOK(true_sv) && BOOL_INTERNALS_sv_isbool(true_sv)
# ok: SvBoolFlagsOK(false_sv) && BOOL_INTERNALS_sv_isbool(false_sv)
# ok: SvBoolFlagsOK(true_sv) && BOOL_INTERNALS_sv_isbool_true(true_sv)
# ok: SvBoolFlagsOK(false_sv) && BOOL_INTERNALS_sv_isbool_false(false_sv)
# ok: SvBoolFlagsOK(true_sv) && !BOOL_INTERNALS_sv_isbool_false(true_sv)
# ok: SvBoolFlagsOK(false_sv) && !BOOL_INTERNALS_sv_isbool_true(false_sv)
# ok: SvTRUE(true_sv)
# ok: !SvTRUE(false_sv)
# Testing 'true_sv_set_bool and false_sv_set_bool'
# ok: SvCUR(true_sv) == 1
# ok: SvCUR(false_sv) == 0
# ok: SvLEN(true_sv) == 0
# ok: SvLEN(false_sv) == 0
# ok: SvIV(true_sv) == 1
# ok: SvIV(false_sv) == 0
# ok: SvIsCOW(true_sv)
# ok: SvIsCOW(false_sv)
# ok: strEQ(SvPV_nolen(true_sv),"1")
# ok: strEQ(SvPV_nolen(false_sv),"")
# ok: SvIOK(true_sv)
# ok: SvIOK(false_sv)
# ok: SvPOK(true_sv)
# ok: SvPOK(false_sv)
# ok: SvBoolFlagsOK(true_sv)
# ok: SvBoolFlagsOK(false_sv)
# ok: SvTYPE(true_sv) >= SVt_PVNV
# ok: SvTYPE(false_sv) >= SVt_PVNV
# ok: SvBoolFlagsOK(true_sv) && BOOL_INTERNALS_sv_isbool(true_sv)
# ok: SvBoolFlagsOK(false_sv) && BOOL_INTERNALS_sv_isbool(false_sv)
# ok: SvBoolFlagsOK(true_sv) && BOOL_INTERNALS_sv_isbool_true(true_sv)
# ok: SvBoolFlagsOK(false_sv) && BOOL_INTERNALS_sv_isbool_false(false_sv)
# ok: SvBoolFlagsOK(true_sv) && !BOOL_INTERNALS_sv_isbool_false(true_sv)
# ok: SvBoolFlagsOK(false_sv) && !BOOL_INTERNALS_sv_isbool_true(false_sv)
# ok: SvTRUE(true_sv)
# ok: !SvTRUE(false_sv)
Failed 1/13 subtests 

Test Summary Report
-------------------
../ext/XS-APItest/t/boolean-thr.t (Wstat: 0 Tests: 5 Failed: 0)
  Parse errors: Bad plan.  You planned 6 tests but ran 5.
../ext/XS-APItest/t/boolean.t    (Wstat: 0 Tests: 12 Failed: 0)
  Parse errors: Bad plan.  You planned 13 tests but ran 12.
Files=2, Tests=17,  0 wallclock secs ( 0.04 usr  0.01 sys +  0.23 cusr  0.05 csys =  0.33 CPU)
Result: FAIL
```
Note that in each test file the final `ok [test number]` statement is not being printed.  I am not observing these test failures with `PERLIO=perlio`.  See, *e.g.,* https://perl5.test-smoke.org/report/5018899.  Also observed on threaded build on FreeBSD-12 and Linux.

**Comment by jkeenan** [comment]
The likely source is this commit from today:
```
commit c5e6a2f4b4dc5540429a0fde0a5b2b01fc167689
Author:     Yves Orton <demerphq@gmail.com>
AuthorDate: Wed Apr 20 12:25:02 2022 +0200
Commit:     Karl Williamson <khw@cpan.org>
CommitDate: Fri May 27 21:18:10 2022 -0600

    XS-APItest - add tests for new API calls and for bool internals
```
@demerphq, @khwilliamson, can you take a look?

**Comment by khwilliamson** [comment]
On 5/28/22 10:01, James E Keenan wrote:
> The likely source is this commit from today:
> 
> |commit c5e6a2f4b4dc5540429a0fde0a5b2b01fc167689 Author: Yves Orton 
> ***@***.***> AuthorDate: Wed Apr 20 12:25:02 2022 +0200 Commit: 
> Karl Williamson ***@***.***> CommitDate: Fri May 27 21:18:10 2022 
> -0600 XS-APItest - add tests for new API calls and for bool internals |
> 
> @demerphq <https://github.com/demerphq>, @khwilliamson 
> <https://github.com/khwilliamson>, can you take a look?
> 

Please check that it is fixed as of d3faea291a338406bbae0e506d519f8ea3fa600e

**Comment by jkeenan** [comment]
> On 5/28/22 10:01, James E Keenan wrote: The likely source is this commit from today: |commit [c5e6a2f](https://github.com/Perl/perl5/commit/c5e6a2f4b4dc5540429a0fde0a5b2b01fc167689) Author: Yves Orton ***@***.***> AuthorDate: Wed Apr 20 12:25:02 2022 +0200 Commit: Karl Williamson ***@***.***> CommitDate: Fri May 27 21:18:10 2022 -0600 XS-APItest - add tests for new API calls and for bool internals | @demerphq <https://github.com/demerphq>, @khwilliamson <https://github.com/khwilliamson>, can you take a look?
> Please check that it is fixed as of [d3faea2](https://github.com/Perl/perl5/commit/d3faea291a338406bbae0e506d519f8ea3fa600e)

Yes, it does.  Confirmed on FreeBSD-12 and -13.  Closing.
